packages/app/components/constants.ts

```diff
@@ -45,3 +45,17 @@ export const getAbiFromJson = (json: {
     stateMutability: a.stateMutability as StateMutabilityType,
     type: a.type as AbiType,
   }));
+
+/*
+ * Config needed to make a firebase DB connection.
+ * TODO: use a Cabin account.
+ */
+export const firebaseConfig = {
+  apiKey: "AIzaSyBlP4xTTtpua7u2Jl0Bpybn896NNNUFwx8",
+  authDomain: "passports-339418.firebaseapp.com",
+  projectId: "passports-339418",
+  storageBucket: "passports-339418.appspot.com",
+  messagingSenderId: "198686036255",
+  appId: "1:198686036255:web:47b24729c8c3e52eaf202a",
+  measurementId: "G-ZS4NL2R3GS"
```

Comment:
These apikeys are now compromised and can't be used. we should set up environment variables to use here instead and we could set up secrets in Vercel

Reply:
we will also want this to be an api route in the `pages/api` folder, because leaking these keys to the front end is also a security vulnerability

Reply:
Done. I added api routes for all the firebase calls and we're now reading the api key from env vars.